### PR TITLE
Add animated component.

### PR DESCRIPTION
**Pull request by adamjspooner**

### CODE REVIEW

**Comment by adamjspooner** [comment]
@keybase/react-hackers, this is ready for review.

**Comment by chrisnojima** [comment]
Please don't merge before we discuss next week

File: shared/package.json [hunk 1]
```diff
@@ -161,6 +161,7 @@
     "react-native-push-notification": "git://github.com/keybase/react-native-push-notification#keybase-fixes-off-311-fcm",
     "react-navigation": "git://github.com/keybase/react-navigation#keybase-fixes-off-beta-12",
     "react-redux": "6.0.0",
+    "react-spring": "^7.2.11",
```
Comment (chrisnojima) [inline shared/package.json]: lets use the exact semver always

Comment (adamjspooner) [reply shared/package.json]: Done. 👍

**Comment by chrisnojima** [comment]
- [ ] after release

**Pull request merged**